_test/relayer_chain_test.go

```diff
@@ -119,7 +120,7 @@ func chainTest(t *testing.T, tcs []testChain) {
 
 	t.Log("Starting relayer")
 	filter := relayer.ChannelFilter{}
-	_ = relayer.StartRelayer(ctx, log, src, dst, filter, 2*cmd.MB, 5)
+	_ = relayer.StartRelayer(ctx, log, src, dst, os.Stdin, os.Stdout, filter, 2*cmd.MB, 5, true, 20)
```

Comment:
Tests should rarely if ever directly interact with `os.Stdin` and `os.Stdout`. 

`strings.NewReader("")` or `bytes.NewReader(nil)` are simple ways to create an empty input. 

`io.Discard` is the idiomatic way to discard writes.

It looks like those are getting passed down to the cosmos client, which I assume will not actually need to interact with those in test.

Reply:
done